django/contrib/gis/db/backends/base/operations.py

```diff
@@ -104,7 +104,12 @@ def get_geom_placeholder(self, f, value, compiler):
 
     def check_aggregate_support(self, aggregate):
         if aggregate.contains_aggregate == 'gis':
```

Comment:
This is only ever a bool, not a string. Unless there's something else in this patch that assigns the string gis to this property?